deploy/olm-catalog/mobile-developer-console-operator/mobile-developer-console-operator.package.yaml

```diff
@@ -0,0 +1,4 @@
+packageName: integreatly-mobile-developer-console
+channels:
+  - name: integreatly
```

Comment:
Let's have the channel name be `stable` here in this project. In the package file in the integreatly manifests, you can call it `integreatly`.